### DISCUSSION TITLE
Problems with work, after disconnecting outdated SSL/TLS protocols

### DISCUSSION THREAD

**Discussion by freefox20000** [lithnet/access-manager#92] [General]
To ensure security, we have disabled outdated encryption protocols on a server with LithNet Access Manager installed. After that, access from all browsers except Internet Explorer stopped working. Access to other resources did not stop.
After returning to its original state, work resumed.
Settings are shown in the screenshot in the attachment
![tls](https://user-images.githubusercontent.com/32125848/114088766-092bb680-98be-11eb-8093-2933943f0d42.jpg)

**Comment by ryannewington** [comment]
Schannel settings apply to the TLS session, which is managed by the Windows web server engine, and kicks in well before the web request reaches access manager. 

The only explanation that I can think of for what you are seeing is that the clients you are using must not be able to negotiate a TLS connection with the combination of ciphers you've enabled.
